src/Filter/FilterEnabledInterface.php

```diff
@@ -25,9 +25,13 @@ interface FilterEnabledInterface extends FilterProviderInterface
      * </code>
      *
      * @param string $name Index in the composite
-     * @param callable|FilterInterface $filter
+     * @param callable(mixed): mixed|FilterInterface $filter
```

Comment:
Same problem here with the `$filter` docs - the callable signature needs parenthesis and correcting to `callable(string, ?object):bool`